share/spice/libraries/libraries.js

```diff
@@ -0,0 +1,75 @@
+(function(env) {
+    "use strict";
+    var search_items = DDG.get_query().split(" ");
+
+    function use_this(search_query) {
```

Comment:
Perhaps I'm confused, but don't we need to verify results coming from the perl side to the js side?